packages/runtime/container-runtime/src/compatUtils.ts

```diff
@@ -150,6 +150,9 @@ const runtimeOptionsAffectingDocSchemaConfigMap = {
 		// Although sweep is supported in 2.x, it is disabled by default until compatibilityVersion>=3.0.0 to be extra safe.
 		"3.0.0": { enableGCSweep: true },
 	} as const,
+	createBlobPlaceholders: {
+		"1.0.0": false,
+	},
```

Comment:
@scottn12 wanted to verify if this looks right to you - should be disabled by default in all cases currently.  Later we might come back and add the version where it should enable by default, but we don't know when that will be yet.